tests/unit/amo/api/test_users.js

```diff
@@ -30,16 +38,46 @@ describe(__filename, () => {
         })
         .returns(mockResponse());
 
-      return currentUserAccount({ api: state.api })
-        .then(() => {
-          mockApi.verify();
-        });
+      await currentUserAccount({ api: state.api });
+      mockApi.verify();
+    });
+  });
+
+  describe('editUserAccount', () => {
+    const mockResponse = (newParams) => createApiResponse({
+      jsonData: createUserAccountResponse(newParams),
     });
 
-    it('throws an error if api state is missing', () => {
-      expect(() => {
-        currentUserAccount({});
-      }).toThrowError(/api state is required/);
+    const getParams = (params = {}) => {
```

Comment:
This is just a nit; take it or leave it. Considering that there's only one test that makes use of it, this `getParams` function is kind of unnecessary. It does seem like a good model to follow, and I know we use it in other test files.  I notice the same model is followed for the other tests in this file, which also don't really need it. If  more tests are added later it would be useful, but at this point none of these cases seems to need more tests.